src/views/AboutView.vue

```diff
@@ -8,35 +8,45 @@
       height="100"
     />
     <h1>Scale Workshop 2</h1>
-    <p><em>Because there are more than 12 notes</em></p>
-    <strong>Contributors</strong>
-    <p>Sevish, Lumi Pakkanen, Lajos Mészáros, Vincenzo Sicurella</p>
+    <p style="margin-bottom: 2rem">
+      <em>Because there are more than 12 notes</em>
+    </p>
     <hr />
-    <ul>
-      <li>
-        <a href="https://github.com/xenharmonic-devs/scale-workshop">GitHub</a>
-      </li>
-      <li>
-        <a href="https://github.com/xenharmonic-devs/scale-workshop/issues"
+    <div class="control-group">
+      <div class="control">
+        <p>
+          With contributions from Sevish, Lumi Pakkanen, Lajos Mészáros,
+          Vincenzo Sicurella
+        </p>
+      </div>
+    </div>
+    <div class="control-group">
+      <div class="control">
+        <a class="btn" href="https://github.com/xenharmonic-devs/scale-workshop">GitHub</a>
+        <a
+          class="btn"
+          href="https://github.com/xenharmonic-devs/scale-workshop/issues"
           >Report issue</a
```

Comment:
*Report an issue* would be more grammatically correct, but might look verbose on a button.